setup/providers/nordvpn.md

```diff
@@ -46,6 +46,7 @@ services:
 
 ### Wireguard only
 
+- `VPN_TYPE`
```

Comment:
Since this only applies to the docker-compose example given (docker run commands have VPN_TYPE mentioned), let's just add an env variable in the yml syntax instead, and remove `VPN_TYPE` from the compulsory variables section:

```
      - VPN_TYPE=openvpn # or wireguard
```

Reply:
Hopefully I've done what you had in mind, let me know if not.